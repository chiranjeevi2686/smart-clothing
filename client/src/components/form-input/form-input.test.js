import React from "react";
import { render } from '@testing-library/react';
import FormInput from './form-input.component';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';

const defaultStore = 
    {
        name:'email',
        type:'email',
        value:'test@email.com',
        handleChange:'true',
        label:'email'
    };
const mockStore = configureStore();
describe('FormInput component', () => {
    it('should render FormInput', async () => {
        const { asFragment } = render(<Provider store={mockStore(defaultStore)}>
            <FormInput />
        </Provider>)
        expect(asFragment()).toMatchSnapshot();
    })
})
