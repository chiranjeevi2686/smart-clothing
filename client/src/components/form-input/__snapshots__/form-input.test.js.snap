// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormInput component should render FormInput 1`] = `
<DocumentFragment>
  <div
    class="sc-bdfBwQ itMfvw"
  >
    <input
      class="sc-gsTCUz kwIyCI"
    />
  </div>
</DocumentFragment>
`;
